### ISSUE TITLE
🐛 [Bug] Theme state (Dark mode) is not saved permanently

### ISSUE THREAD

**Issue by ulfgebhardt** [dreammall-earth/dreammall.earth#1827]
## 🐛 Bug

Theme state (Dark mode) is not saved permanently

![image](https://github.com/user-attachments/assets/a10cfe81-a105-4c01-adf2-f1bbc7c39163)

After reload:

![image](https://github.com/user-attachments/assets/778d5b20-addd-408b-9a24-a03fd234ab6a)

It might be wise to save this state just in a cookie and not on the server.

**Comment by Bettelstab** [comment]
I agree, but this means we need to deactivate SSR completely (#1826 ) (fine for me) or manage to read the cookie value on the server and act accordingly. (Possible, but potentially many hours of work)
